api/sessionrecording/gzip_reader.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2024 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sessionrecording
+
+import (
+	"compress/gzip"
+	"io"
+
+	"github.com/gravitational/trace"
+)
+
+// gzipReader wraps file, on close close both gzip writer and file
```

Comment:
```suggestion
// gzipReader wraps an io.Reader, and closes both the gzip writer and underlying reader
```

I know this godoc existed before, but we might as well improve it while we're here.

There's no requirement for a "file" anywhere in this API.

Reply:
Updated this godoc and then ended up removing this file entirely.